build.sbt

```diff
@@ -62,3 +72,16 @@ lazy val allProjects: Seq[ProjectReference] = Seq(
   `memeid4s-http4s`,
   `memeid4s-scalacheck`
 )
+
+lazy val micrositeSettings: Seq[Def.Setting[_]] = Seq(
```

Comment:
These settings could be extracted to an `AutoPlugin` inside `project` folder that automatically activates for any project enabling both `MdocPlugin` and `MicrositesPlugin`:

```scala
import mdoc.MdocPlugin
import microsites.MicrositesPlugin
import microsites.MicrositesPlugin.autoImport._
import com.typesafe.sbt.site.SitePlugin.autoImport._
import sbt.Keys._
import sbt._

object WebsitePlugin extends AutoPlugin {

  override def requires: Plugins = MdocPlugin && MicrositesPlugin

  override def trigger: PluginTrigger = allRequirements

  override def projectSettings: Seq[Def.Setting[_]] = Seq(
    micrositeName                 := "memeid",
    micrositeDescription          := "A JVM library for generating RFC-compliant Universal Unique Identifiers (UUIDs)",
    micrositeBaseUrl              := "memeid",
    micrositeDocumentationUrl     := "docs",
    micrositeGithubOwner          := "47degrees",
    micrositeGithubRepo           := "memeid",
    micrositeOrganizationHomepage := "https://www.47deg.com",
    micrositePushSiteWith         := GitHub4s,
    micrositeHighlightLanguages   ++= Seq("java", "scala", "kotlin"),
    includeFilter in makeSite     := "*.html" | "*.css" | "*.png" | "*.jpg" | "*.gif" | "*.js" | "*.swf" | "*.md" | "*.svg"
  )

}
```

Reply:
Beautiful!